code/model/MultiForm.php

```diff
@@ -176,6 +176,15 @@ public function getController() {
 	}
 
 	/**
+	 * Returns the get_var to the template engine
+	 *
+	 * @return string
+	 */
+	public function get_var() {
```

Comment:
Is this the proper naming convention for public methods? It should be lower camel case (psr-2 and standard ss convention).